packages/core/src/radio-group/radio-group.ts

```diff
@@ -0,0 +1,122 @@
+import type { CSSResultGroup, TemplateResult } from 'lit';
+import type { Radio } from './radio.js';
+import { FormControlMixin } from '@open-wc/form-control';
+import { LitElement, html } from 'lit';
+import { property, queryAssignedElements } from 'lit/decorators.js';
+import styles from './radio-group.scss.js';
+
+const OBSERVER_OPTIONS: MutationObserverInit = {
+  attributeFilter: ['checked'],
+  attributeOldValue: true,
+  subtree: true
+};
+
+export class RadioGroup extends FormControlMixin(LitElement) {
```

Comment:
Maybe radiogroup should get role `radiogroup`?

https://www.w3.org/WAI/ARIA/apg/patterns/radiobutton/ 
"The radio buttons are contained in or owned by an element with role [radiogroup](https://w3c.github.io/aria/#radiogroup)."